invenio_communities/receivers.py

```diff
@@ -27,3 +29,14 @@ def inject_provisional_community(sender, json=None, record=None):
     q = InclusionRequest.query.filter_by(id_record=record.id)
     provisional_communities_ids = [r.id_community for r in q]
     json['provisional_communities'] = provisional_communities_ids
+
+
+def create_oaipmh_set(mapper, connection, community):
+    """Signal for creating OAI-PMH sets during community creation."""
+    from invenio_oaiserver.models import OAISet
+    with db.session.begin_nested():
+        obj = OAISet(spec=community.id,
+                     name=community.title,
+                     description=community.description,
+                     search_pattern="community:{0}".format(community.id))
```

Comment:
`search_pattern='community:"{0}"'.format(community.id))` (note the double quotes)